### PR TITLE
Install and config pundit

### PR DESCRIPTION
**Pull request by edkajimenez**
## Summary
* Install pundit
* Config Pundit
* Create rent policy
* Edit tests

## Trello Card
https://trello.com/c/SYdiZDYQ/12-instalar-y-configurar-pundit

### CODE REVIEW

File: config/application.rb [hunk 1]
```diff
@@ -23,6 +23,7 @@ class Application < Rails::Application
     # if a locale isn't found fall back to this default locale
     config.i18n.fallbacks = true
     # set the possible locales to English, Portuguese and Spanish
-    config.i18n.available_locales = [:en, :pt, :es] 
+    config.i18n.available_locales = [:en, :pt, :es]
```
Comment (mtejedorwolox) [inline config/application.rb:26]: checkout spaces and identation here, this line didn't change

File: spec/controllers/rents_controller_spec.rb [hunk 2]
```diff
@@ -72,6 +70,19 @@
         end
       end
     end
+
+    context 'When unauthorized user' do
+      let(:book)      { create(:book) }
```
Comment (mtejedorwolox) [inline spec/controllers/rents_controller_spec.rb:75]: i think you can do build(:book) and build(:user), check it

File: spec/controllers/rents_controller_spec.rb [hunk 3]
```diff
@@ -99,17 +111,16 @@
       end
 
       it 'responds with all the rents' do
-        expected = rents.count
-        expect(response_body['page'].length).to eq expected
+        expected = rents.map { |rent| rent.user == user }
```
Comment (mtejedorwolox) [inline spec/controllers/rents_controller_spec.rb]: You are assigning the same user to all the rents, so this mapping it's useless

**Review by mtejedorwolox — APPROVED** [review]

File: config/application.rb (hunk 1 above)
Comment (edkajimenez) [reply config/application.rb:26]: It seems to be OK, no idea why is showing a change

File: spec/controllers/rents_controller_spec.rb (hunk 2 above)
Comment (edkajimenez) [reply spec/controllers/rents_controller_spec.rb:75]: No, I tested it, it is needed book and user to exist to be able to create a rent

**Pull request merged**